core/server/worker/src/main/java/alluxio/web/WorkerWebServer.java

```diff
@@ -47,6 +47,8 @@
   public static final String ALLUXIO_FILESYSTEM_CLIENT_RESOURCE_KEY =
       "Alluxio Worker FileSystem Client";
 
+  private FileSystem mFileSystem;
```

Comment:
What do you think of placing this in `WebServer`?

Reply:
I considered it, but because the JobMaster/WorkerWebServer wouldn't use it I didn't feel it was appropriate to put it there.